### PR TITLE
put all obsolete files in folder und clean up my code

### PR DESCRIPTION
**Pull request by martinhaiss**
I created a folder "obsolete files" and put files in it which are most likely no longer needed. Also cleaned up parts of my code.

### CODE REVIEW

File: src/components/cards/Cards.js [hunk 1]
```diff
@@ -1,16 +1,17 @@
...
+// import CardHermione from "../CardHermione/CardHermione";
+// import CardCedric from "../CardCedric/CardCedric";
+// import CardDraco from "../CardDraco/CardDraco";
+// import CardHarry from "../CardHarry/CardHarry";
+// import CardRon from "../CardRon/CardRon";
+// import harryPicture from "../img/harry.png";
+// import dracoPicture from "../img/draco.png";
+// import hermionePicture from "../img/hermione.png";
+// ((import cedricPicture from "../img/cedric.png";
+// import ronPicture from "../img/ron.png";
```
Comment (macfrei) [inline src/components/cards/Cards.js:11]: Please remove this commented out code - you don't use it anymore 🙂

File: src/components/cards/Cards.js [hunk 2]
```diff
@@ -30,7 +31,8 @@ function Cards() {
 
 export default Cards;
 
-/* --- old cards
+/* ------ former "cards.js" before import via "data.json" --------
+
 
 function Cards() {
   return (
```
Comment (macfrei) [inline src/components/cards/Cards.js:38]: I do understand that for reference it is nice to still have this commented out code, however, this is what git is best at! Even if you remove it, you have commited it and you can check it out in one of your previous commits. This way you can keep you main branch clean of all commented out code.

**Review by macfrei — COMMENTED** [review]
I added a few comments, Martin. Overall it looks good! 
One thing I couldn't comment on is your `obsolete files` folder. I have two things to say:
1. Folder names should _neve_ be with a space in the name. When importing or referencing to them, there is a chance that the computer gets it wrong, better would be: `obsoleteFiles`.
2. Git saves those changes you have made in commits - so if you want to check out how you have done things earlier, you can check out the commit. You don't need to save obsolete files - and you shouldn't either. 
I know for this it's not critical as it is a repository to get to know react. I just want to make you aware that this is not considered best practice 🙂. Hope this is okay, please get back to me if you have any question!

File: src/components/cards/Cards.js (hunk 1 above)
Comment (martinhaiss) [reply src/components/cards/Cards.js:11]: Thx for the heads up!

ad1) I actually know, but still done wrong... 🙈

ad2) I totally agree. I just wasn't sure if the files were needed for todays class and wanted to be able to restore them in seconds if needed. Same with the commented out code. I'll delete/remove both ✌️🙂

**Pull request merged**